### DISCUSSION TITLE
Scope Important Configuration Selector to Plugin Components

### DISCUSSION THREAD

**Discussion by pcbowers** [tailwindlabs/tailwindcss#7393] [Help]
As I was working with the daisyui component kit, I noticed that the `important` configuration was not working as expected. While all my tailwind utility classes were being scoped appropriately, the components from this plugin were not. After scrolling through the Tailwind CSS documentation, I noticed the following information that mentioned that components are not affected by the users `important` preference (see https://tailwindcss.com/docs/plugins#prefix-and-important-1). Is there anyway around this?

Here is a play that illustrates my issue: https://play.tailwindcss.com/RbsGkkGP86. I would like to still use the plugin components, but I need them scoped just to the #app selector. Is there any other way to do this since it is not supported by the `important` configuration?

**Comment by dohomi** [comment]
@pcbowers I came across this very issue as well. Do you have any solution for this?

**Comment by sfinktah** [comment]
It also says "By default, plugin utilities automatically respect the user’s and important preferences." 
(see https://tailwindcss.com/docs/plugins#prefix-and-important)

i.e. `important` will work for [plugin] utilities, but not for components.

unfortunately i do not yet know how to delineate the two, though my issue is also with integrating dasiyui.  i will advise with updates, if progress is made.

**Comment by sfinktah** [comment]
Progress has not been made, and as I return to the world of tailwindcss I find myself extremely annoyed that a google search on the problem reveals only my confused self from 18 months ago, and little else.
